home_work_converter/Converter.h

```diff
@@ -0,0 +1,80 @@
+//
+// Created by flyon21 on 07.12.24.
+//
+
+#ifndef C_PRO_CONVERTER_H_converter
+#define C_PRO_CONVERTER_H_converter
+
+#include <iostream>
+#include <string>
+#include <stdexcept>
+#include <vector>
+
+enum class ConversionType { Length, Mass, Temperature };
+
+template <ConversionType Type>
+struct ConversionFactor {
+    const char* from;
```

Comment:
probably here would be better to set from to null because other developer could reuse in n the wrong way
const char* from = nullptr;